runner/src/examples.rs

```diff
@@ -558,6 +558,59 @@ fn run_functions_example(example: &FunctionsExample) -> Step {
     }
 }
 
+pub fn build_functions_example(opt: &RunFunctionsExamples) -> Step {
+    let example: Example = example_toml_files()
+        .map(|path| {
+            toml::from_str(&read_file(&path)).unwrap_or_else(|err| {
+                panic!("could not parse example manifest file {:?}: {}", path, err)
+            })
+        })
+        .find(|example: &Example| match &opt.example_name {
+            Some(example_name) => &example.name == example_name,
+            None => true,
+        })
+        .expect("could not find the specified example");
+
+    // build any backend server
```

Comment:
This comment mentions "backend server", but the code is all about clients. Also, the comment should be a sentence that starta with a capital and enda with a full stop.

Reply:
Fixed.